src/Search.php

```diff
@@ -791,8 +791,15 @@ public static function constructSQL(array &$data)
        //// 3 - WHERE
 
        // default string
-        $COMMONWHERE = self::addDefaultWhere($data['itemtype']);
-        $first       = empty($COMMONWHERE);
+        if (
+            Session::getCurrentInterface() != 'helpdesk'
+            && $data['itemtype'] == Ticket::class
+        ) {
```

Comment:
It means that you remove all default checks on every itemtype except `Ticket`. This seems dangerous to me.

Reply:
I agree, not sure how it be fixed tbh
The criteria use the search engine, which applies the defaultWhere for user.
We can't do much about it IMO.

Looking at the default where for User, all it does is add entities restriction.
Is it really a bug ? Should you see users outside your entities ?

Reply:
Is this normal to be able to assign users that will not be visible in the corresponding ticket scope ? Unless I miss something, either the actor dropdown is too permissive, either the search results are too restrictives.

But, honestly, I do not know what should be done here. The `Search::giveItem()` will only show the user name, anonymized if this feature is enabled, and without any tooltip. I guess it is not a big deal to show names of users that are actors of a ticket you are able to see. My only concern is about the API. I do not know what information could be available if we lower the restrictions for the simplified interface.

Anyway, if something specific has to be done for the simplified interface, then it should probably be done in the `Search::addDefaultWhere()` method for this specific `User` case.